examples/example_utils.hpp

```diff
@@ -30,7 +29,8 @@ struct ExampleUtils
 
 mip::Timestamp getCurrentTimestamp();
 
-std::unique_ptr<ExampleUtils> openFromArgs(const std::string& port_or_hostname, const std::string& baud_or_port, const std::string& binary_file_path);
+std::unique_ptr<ExampleUtils> openFromArgs(const std::string& port_or_hostname, const std::string& baud_or_port,
+    const std::string& binary_file_path);
 
```

Comment:
What do you think about using an "all or none" policy for splitting arguments? As in all the args go one line or all on their own line?